Update transaction utility function to use the new API client - Closes #4485

reyraa opened the pull request:
### What was the problem?

This PR resolves #4485

### How was it solved?
- [x] Mock account retrieval to use temporarily
- [x] Fix address derivation path
- [x] Clean up the transaction utility functions  (remove redundant parameters)
- [x] Update actions to pass minimum necessary arguments
- [x] Update parameters of token transfer tx.
- [x] Update broadcast endpoint
- [x] Update multsig signing actions, utils, and components.
- [x] Make sure all tx types work
- [x] Update unit test

**Note:**
1. The changes in this PR do not resolve issues mentioned in #4497 and #4506.
2. You need to set your network to run against a local custom node (lisk-sdk v6.0.0-alpha.4)
3. You need to ensure the `useLegacyAccount` hook returns all required account data.
4. You need to ensure API calls are all made to your local node.
5. You should install `@liskhq/lisk-client@6.0.0-alpha.3`

### How was it tested?
- [x] Test all tx types manually
- [x] Update unit tests

reyraa committed:
Update tx utilities to work with SDK v6 changes

reyraa committed:
Update other transaction actions to match the function signature

reyraa committed:
Remove default derivation path

reyraa committed:
Update multisig signing process accordingly

reyraa committed:
Update delegate registration keys

reyraa committed:
Account for wallets without a lockedBalance

reyraa committed:
Update multisignature registration moduleCommand name and params

reyraa committed:
Fix broken multisig memebr signature

reyraa committed:
Fix voting process

ikem-legend committed:
Fixed SendSummary and VoteSummary tests

reyraa committed:
Fix unlock tx

ikem-legend committed:
Fixed DeepScan and lint errors

ikem-legend committed:
Fixed some DeepScan issues

ikem-legend committed:
Fixed failing tests

ikem-legend committed:
Fixed transaction summarizer test and minor changes

reyraa committed:
Add TODO comments for issues to fix later

reyraa committed:
Update tx signing utils

reyraa committed:
Fix DeepScan issues

reyraa committed:
Fix failing unit tests $1

reyraa committed:
Fix failing unit tests $2

reyraa committed:
Update mock tokenID

ikem-legend committed:
Fixed network related test failures and minor changes

reyraa committed:
Fix failing unit tests $3

reyraa committed:
Fix failing unit tests $4

ManuGowda commented on src/const/regex.js:
```diff
@@ -3,10 +3,10 @@ export const regex = {
   proofOfPossession: /^[0-9a-f]{192}$/,
-  username: /^[a-z0-9!@$&_.]{3,20}$/,
+  name: /^[a-z0-9!@$&_.]{3,20}$/,
   delegateName: /^[a-z0-9!@$&_.]{3,20}$/,
```
i think the username regex was for account creation and addition flow? since the logic is same `delegateName` as well, either we can reuse `name` regex for validating the account name as well? and remove `delegateName` as well?

ManuGowda commented on src/modules/dpos/validator/components/RegisterDelegateForm/index.js:
```diff
@@ -63,12 +62,12 @@ const RegisterDelegateForm = ({
   const transaction = {
     moduleCommand: MODULE_COMMANDS_NAME_MAP.registerDelegate,
     params: {
```
we need to add param `"delegateRegistrationFee": "1000000000"` for now you can hardcode the value `1000000000` and add todo, once service exposes the constants endpoint we can integrate it.

reyraa committed:
Fix sender index lookup

soroushm committed:
fix: unit test

ManuGowda reviewed (dismissed):
Tested following types of transactions:
- Token transfer
- Multisig registration (not working, but worked for Ali)
- Delegate registration
- Voting

Overall code changes are ok considering the issue objective. We can do further refactoring in next issues.

Commit:
:recycle: Fix failing unit tests
Updated command parameter schemas
Update API version from v2 to v3
Resolved haning tests by correct mocking

Commit:
:recycle: Update CI to skip core readiness check

Commit:
:recycle: Update CI to skip core readiness check

soroushm commented:
LGTM but it need adjust Manu comments

Commit:
:recycle: Update CI to skip core and service deployment
